tools/update-webpage.sh

```diff
@@ -44,6 +45,7 @@ titles[$REFERENCE_COUNTING_MD]="Reference Counting"
 titles[$DEBUGGER_MD]="Debugger"
 titles[$CODING_STANDARDS_MD]="Coding Standards"
 titles[$EXT_REFERENCE_ARG_MD]="'Extension API: Argument Validation'"
+titles[$EXT_REFERENCE_AUTORELEASE_MD]="'Extension API: Autorelease values'"
```

Comment:
Please start every word with a capital letter like the others: `Autorelease Values`